code/modules/power/data_terminal.dm

```diff
@@ -0,0 +1,29 @@
+//I promise that this isn't just 100% stolen from goon. Honest~ -Francinum
+
+/obj/machinery/power/data_terminal
+	name = "data terminal"
+	icon_state = "dterm"
+	use_data = TRUE
+	layer = LOW_OBJ_LAYER
+	var/obj/machinery/connected_machine // Whatever machine that got built over us, We proxy it's packets back and forth.
+
+/obj/machinery/power/data_terminal/Initialize(mapload)
+	. = ..()
+	connect_to_network()
```

Comment:
Destroy()?

Reply:
Handled by machinery/power's destroy. I have no idea why it doesn't inherently try and connect automatically, but I'm also not in the mood to dick with MORE ancient shitcode outside the scope of this PR.